src/en/weebdex/src/eu/kanade/tachiyomi/extension/en/weebdex/dto/ChapterDto.kt

```diff
@@ -0,0 +1,35 @@
+package eu.kanade.tachiyomi.extension.en.weebdex.dto
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ChapterListDto(
```

Comment:
Remove `data` modifier unless needed
```suggestion
class ChapterListDto(
```